docs/99-references/00-references.adoc

```diff
@@ -13,20 +13,49 @@ Dieser Abschnitt enthält Quellenangaben, die ganz oder teilweise im Curriculum
 This section contains references that are cited in the curriculum.
 // end::EN[]
```

Comment:
You have so many references but you barely reference any of them.

Reply:
I'm aware, but IMHO rectifying this would warrant a new version of the module.

Reply:
Fair enough.